### DISCUSSION TITLE
Doorbell zones on Frigate. Can I somewhat rely on it as a car theft alarm at night?

### DISCUSSION THREAD

**Discussion by Chaoscontrol** [blakeblackshear/frigate#13832] [Ask A Question]
I just got a new car, and would love to set some sort of warning if someone messes with it at night. Nothing too fancy. Just being aware. 

I'm worried about setting the zone shown in Frigate. When the car is not there, passer-by or next door neighbours can easily get in that zone. Even when the car is there, neighbours might need to pass next to the car to go to their bins or back garden. 

Anyway. Regarding the zone, what else apart from the zone would be good to add to the config so it triggers detection/alerts when people are around the car, but doesn't when it's more in the background?

Thanks!

![image](https://github.com/user-attachments/assets/716d004d-1e0b-4137-9335-697d98e614ab)

**Comment by hawkeye217** [comment]
Remember that an object is considered "in a zone" when [the bottom center of the object's bounding box is in the zone](https://docs.frigate.video/configuration/zones/). 

If you want to catch people around the car, the bounding box of a person may only be surrounding their torso as the fence obscures their feet and legs. The`Driveway/Neighbours` zone may or may not catch them, depending on where the bounding box lands. I'd experiment and watch the debug view, but you may want to move the edge of the zone down slightly along the fence.

**Reply by Chaoscontrol** [reply]
Good point, will add some margin to them through the bottom.

**Comment by hawkeye217** [comment]
You could also consider setting the `loitering_time` for a zone. If someone is trying to open your car door or is peering inside, they're more likely to be there for more time than someone just passing through.

**Reply by Chaoscontrol** [reply]
Good one as well. Thanks. 

What about my worry of background stuff creating false positives? Any way to reduce/prevent that?

**Reply by hawkeye217** [reply]
You could perhaps set a `min_area` filter so you aren't picking up a small `person` that's so far away.
